docs/installation/introduction.rst

```diff
@@ -1,6 +1,104 @@
 Introduction
 ============
 
+Foreword
+^^^^^^^^
+MetalK8s is a Kubernetes_ distribution with a number of addons carefully
+picked for optimal on-premises deployments, including pre-configured monitoring
+and alerting, self-healing system configuration, and more.
+
+The installation of a MetalK8s cluster can be broken down into
+the following steps:
+
+#. :doc:`Setup <./setup>` of the environment (with requirements and example
+   OpenStack deployment)
+#. :doc:`Deployment <./bootstrap>` of the :term:`Bootstrap node`, the first
+   machine in the cluster
+#. :doc:`Expansion <./expansion>` of the cluster, orchestrated from the
+   Bootstrap node
+
+.. _Kubernetes: https://kubernetes.io/
+
+.. _installation-intro-architecture:
+
+Choosing a Deployment Architecture
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Before starting the installation, choosing an architecture is recommended, as
+it can impact sizing of the machines and other infrastructure-related details.
+
+.. note:: "Machines" may indicate bare-metal servers or VMs interchangeably
+
+Standard Architecture
+"""""""""""""""""""""
+MetalK8s has been designed with on-premise, offline, reliable deployments in
```

Comment:
Since we should be talking about the present(now) and presumably looking into the future, knowing that we will continue to follow these design pattern; this could read...

`MetalK8s is designed with on-premise, offline ....`

No??